sdk/tables/azure-data-tables/README.md

```diff
@@ -177,21 +161,34 @@ The following sections provide several code snippets covering some of the most c
 
 
 ### Creating a table
-Create a table in your account
+Create a table in your account and get a `TableClient` to perform operations on the newly created table:
 
 ```python
 from azure.data.tables import TableServiceClient
 table_service_client = TableServiceClient.from_connection_string(conn_str="<connection_string>")
-table_service_client.create_table(table_name="myTable")
+table_name = "myTable
+table_service_client.create_table(table_name=table_name)
+table_client = table_service_client.get_table_client(table_name=table_name)
 ```
 
 ### Creating entities
-Create entities in the table
+Create entities in the table. The properties
 
 ```python
 from azure.data.tables import TableServiceClient
-
-my_entity = {'PartitionKey':'part','RowKey':'row'}
+from datetime import datetime
+
+my_entity = {
+    'PartitionKey': 'product_name',
+    'RowKey': 'product_id'
```

Comment:
Not sure if the term `product_id` really helps here. Maybe we could really call it out with something like:
```python
PRODUCT_ID = '001234'
PRODUCT_NAME = 'RedMarker'

my_entity = {
    'PartitionKey': PRODUCT_NAME,
    'RowKey': PRODUCT_ID,
    ...
}
```
```